src/lib/components/shared/vercel_hover.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+    export let glider_container_class = "";
+    export let active_element_class = "";
+    export let direction: "horizontal" | "vertical";
+
+    /** vercel effect */
+    let hover_glider_element: HTMLElement,
+        glider_container_element: HTMLElement,
+        GLIDER_TRANSITION_DURATION = 200,
+        is_hovered = false,
+        mouse_leave_timeout: NodeJS.Timer | null = null;
+
+    const handle_mouseenter = (event: MouseEvent) => {
+        const target = event.target as HTMLElement;
+        const target_computed_style = getComputedStyle(target);
+
+        // To make sure our operations are proper we need to make sure that the `position` is set to relative
+        glider_container_element.style.position = "relative";
```

Comment:
Why not use `relative` class?

Reply:
Ease of use and there can be relative passed down from the parent